.travis.yml

```diff
@@ -0,0 +1,23 @@
+env:
+  - CXX=g++-4.8
+
```

Comment:
I had to add these configs.  See: https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements